packages/graphback/tests/buildGraphbackAPI.test.ts

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { createKnexDbProvider, CRUDService } from '@graphback/runtime-knex'
+import * as Knex from 'knex'
+import { buildSchema } from 'graphql'
+import { buildGraphbackAPI } from '../src'
+import { DataSyncPlugin } from '../../graphback-datasync/src/DataSyncPlugin'
+
+describe('buildGraphbackAPI', () => {
+
+  const setup = () => {
+    const db = Knex({
+      client: 'sqlite3',
+      connection: {
+        filename: ':memory:'
+      },
+      useNullAsDefault: true
+    })
+
+    return { db }
+  }
+
+  test('Creates a Graphback API using the default configuration with Knex', () => {
```

Comment:
Tests now added :)

Reply:
Nice. Should the name of the test file be `buildGraphbackAPITest`?

Reply:
I will change it for consistency, I prefer `buildGraphbackAPI.test.ts` though because it is much easier to see at a glance that it is a test. I think it is a more common convention in JS/TS to do `.spec.ts` or `.test.ts`. 

Maybe we will change them all sometime.

![image](https://user-images.githubusercontent.com/11743717/84041880-4375b000-a99c-11ea-84ab-b9dcb9885eb2.png)